tests/test_styles.py

```diff
@@ -74,6 +74,68 @@ def test_style_lhcb():
     return fig
 
 
+@check_figures_equal(extensions=["pdf"])
+# @pytest.mark.parametrize(
+#     "experiment_style",
+#     [hep.style.ALICE, hep.style.ATLAS, hep.style.CMS, hep.style.LHCb, hep.style.ROOT,],
+#     ids=["ALICE", "ATLAS", "CMS", "LHCb", "ROOT"],
+# )
```

Comment:
@andrzejnovak Do you know any clean way to parameterize these types of test? The `mpl` `check_figures_equal` decorator didn't seem to like me trying to apply `@pytest.mark.parametrize`.

Reply:
I'm never sure about the syntax of these things, but this seems to work 
```
import mplhep.style as _sty
@check_figures_equal(extensions=["pdf"])
@pytest.mark.parametrize(
    "plt_style,set_style", 
    list(zip([_sty.ALICE, _sty.ATLAS, _sty.CMS, _sty.LHCb, _sty.ROOT, _sty.ROOT],
             ["ALICE", "ATLAS", "CMS", "LHCb", "ROOT", _sty.ROOT]))
    )
@pytest.mark.skipif(sys.platform != "linux", reason="Linux only")
def test_set_style(plt_style, set_style, fig_test, fig_ref):
    plt.rcParams.update(plt.rcParamsDefault)

    hep.rcParams.clear()
    plt.style.use(plt_style)
    fig_ref.subplots()

    hep.rcParams.clear()
    hep.set_style(set_style)
    fig_test.subplots()
```

Reply:
I got it now. I had just messed up and hadn't been running with `--mpl-generate-path=tests/baseline`